pandas/tests/indexes/test_base.py

```diff
@@ -1142,19 +1106,21 @@ def test_format(self):
             expected = [str(index[0])]
             assert formatted == expected
 
+        self.strIndex[:0].format()
+
+    def test_format_missing(self, nulls_fixture):
```

Comment:
Was only testing `np.nan` and `None` before but figured this was a good use case for the new nulls fixture